README.md

```diff
@@ -63,6 +64,20 @@ java_ks { 'broker.example.com:/etc/activemq/broker.ks':
 }
 ```
 
+For use cases where you hire to fetch the certificate data from a secure store, like vault, you can use the `_content` attributes. Here is an example:
```

Comment:
Should `hire` be something like `want`? For example:

```suggestion
For use cases where you want to fetch the certificate data from a secure store, like vault, you can use the `_content` attributes. Here is an example:
```

Reply:
Unknowingly, Visual Studio code automatically updated the ToC for me :-(. I have removed this now. Also the typo.